commons/addons/mysql.md

```diff
@@ -257,3 +257,11 @@ If you want to import your **SQL** dump, you can use several methods:
 3. Any MySQL client such as [MySQL Workbench](https://www.mysql.fr/products/workbench/).
 
 If you need to import a very large dump, please send an email to <support@clever-cloud.com>.
+
+## Direct access
+
+All our dedicated MySQL database are served via a proxy. To reduce the latency you can by pass this proxy by generating direct hostname and port for the addon. You can do it by clicking the "Generate direct hostname and port" on the addon dashboard.
+
+This action will add new environment variables to reach the addon without proxy.
+
+The usage of direct access is a tradeoff: when you migrate your addon, you will need to generate hostname and port again so your application will need a reapply of these environment while proxy usage changes nothing.
```

Comment:
Keep in mind that usage* of direct access is a trade-off*
will need an update* of these environments*